pkg/clusteragent/admission/mutate/config/mutator.go

```diff
@@ -227,7 +244,7 @@ func injectExternalDataEnvVar(pod *corev1.Pod) (injected bool) {
 	return
 }
 
-func buildVolume(volumeName, path string, hostpathType corev1.HostPathType, readOnly bool) (corev1.Volume, corev1.VolumeMount) {
+func buildHostPathVolume(volumeName, path string, hostpathType corev1.HostPathType, readOnly bool) (corev1.Volume, corev1.VolumeMount) {
```

Comment:
nit: `readOnly` seem to always be `true`, if there is not potential use case of it being `false`, it should probably be removed from the function signature.